homeassistant/components/sensor/const.py

```diff
@@ -561,7 +561,7 @@ class SensorStateClass(StrEnum):
     SensorDeviceClass.ILLUMINANCE: {SensorStateClass.MEASUREMENT},
     SensorDeviceClass.IRRADIANCE: {SensorStateClass.MEASUREMENT},
     SensorDeviceClass.MOISTURE: {SensorStateClass.MEASUREMENT},
-    SensorDeviceClass.MONETARY: {SensorStateClass.TOTAL},
+    SensorDeviceClass.MONETARY: {SensorStateClass.TOTAL, SensorStateClass.TOTAL_INCREASING, SensorStateClass.MEASUREMENT},
```

Comment:
This PR is incorrect; the device class is meant for aggregations, not current values.

Reply:
@frenck thanks for taking the time to look at this PR, however I'm unsure as to why its incorrect.

what device_class and state_class would you suggest is used for the current unit cost of electricity. I would have assumed monetary and measurement based on the docs but that is not currently supported. monetary as it is a monetary value and measurement as it represents the current cost of something (which can and does change from time to time).

`TOTAL_INCREASING` would also seem like a more appropriate state class for the today's total electricity costs rather than `TOTAL` as it only ever increases and it resets each day at midnight.